learn/advanced/faceted_search.md

```diff
@@ -0,0 +1,208 @@
+
+If none of the matching documents have a numeric value for a facet, that facet is not included in the `facetStats` object.
+
+Meilisearch ignores string values like `"21"` when computing `facetStats`.
+
+The following response shows the `facetStats` when searching for `led`:
+
+```json
+{
+  …
+  "facetStats":{
+    "rating":{
+      "min":0.0,
+      "max":5.0
+    }
+  }
+  …
+}  
+```
+
+Since `rating` was the only numeric facet in our example, it is returned in the `facetStats` object.
+
+## Facet types
+
+### Single select facets
+
+Single-select or conjunctive facets use the `AND` logical operator. They only allow the selection of a single value per facet.
+
+Suppose your `ecommerce` index contains a facet for `brand` with the values `BrandA`, `BrandB`, and `BrandC`. Single select facets will allow you to view products from one brand at a time. If you want to view `BrandA` and `BrandB` products, you will have to try both searches individually. If you want to further narrow down results using the `reviews` facet, you would need to do so separately for both brands.
+
+The image below shows single-select facets in action. When the user searches for `led` and selects `Sunlite`, other brands are no longer visible.
+
+![Meilisearch demo for single-select facets searching for 'LED'](/faceted-search/facets-single-select.png)
+
+The code sample below shows the query for the search mentioned above:
+
+<CodeSamples id="faceted_search_conjunctive_facets_1" />
+
+### Multi-select facets
+
+Multi-select or disjunctive facets use the `OR` logical operator. They allow users to choose multiple options within a facet so they don’t have to perform more than one search to find the products they’re looking for.
+
+With single select facets, if a user selects `BrandA` for the `brands` facet, they will see it offers 5 products, nothing more. But with multi-select facets, we want to show users how many products `BrandB` and `BrandC` offer even when they're not selected. This lets the user know if other brands offer similar products and allows them to view products from multiple brands at the same time.
+
+The image below shows multi-select facets in action. Even though the user selected the `Lithonia Lighting` and `Sunlite` brands when searching for `LED`, they can see other brands and how many LED products they have.
+
+![Meilisearch demo for multi-select facets searching for 'LED'](/faceted-search/facets-multi-select.png)
+
+The code sample below shows the query for the search mentioned above:
+
+<CodeSamples id="faceted_search_disjunctive_facets_1" />
+
+To learn more about implementing faceting and filtering, check out our [ecommerce demo](https://github.com/meilisearch/ecommerce-demo).
```

Comment:
@Strift
I linked the GitHub repository here so readers have access to both the repo and demo.

Reply:
Thank you for the heads up!